src/99-appendices/10-file-collections.md

```diff
@@ -24,24 +22,39 @@ included in this appendix:
-| TB1RMF | Please see the qMRI appendix | Siemens `rf_map`                   | B1<sup>+</sup> data acquired using `rf_map` product sequence by Siemens                                                                                                                                                                                                                                                                                                           |
-| RB1COR | Please see the qMRI appendix | B1<sup>-</sup> field correction    | Low resolution images acquired by the body coil (in the gantry of the scanner) and the head coil using identical acquisition parameters to generate a combined sensitivity map as described in [Papp et al. (2016)](https://onlinelibrary.wiley.com/doi/full/10.1002/mrm.26058)                                                                                                   |
+```Text
+sub-<label>/[ses-<label>/]
+    fmap/
+        sub-<label>[_ses-<label>][_acq-<label>][_ce-<label>][_rec-<label>][_run-<index>][_echo-<index>][_flip-<index>][_inv-<index>][_mt-<label>][_part-<label>]_<suffix>.nii[.gz]
+```
+
+| Suffix  | Meta-data relevant entity                                    | Application                        | Description                                                                                                                                                                                                                                                                                                                                                                                                                                             |
+|---------|--------------------------------------------------------------|------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| TB1DAM  | flip                                                         | Double-angle B1+ mapping           | The double-angle B1<sup>+</sup> method ([Insko and Bolinger 1993](https://doi.org/10.1006/jmra.1993.1133)) is based on the calculation of the actual angles from signal ratios, collected by two acquisitions at different nominal excitation flip angles. Common sequence types for this application include spin echo and echo planar imaging.                                                                                                        |
```

Comment:
All of these suffixes should be represented in a suffix table in the field map section of the MRI page, right?

Reply:
@tsalo if you envision that adding them there won't lead to confusion for more common use cases, we could represent them in this table. My concern was that the table is already a bit heterogeneous concerning what `suffix` represents.

Reply:
Hmm... ultimately, we will need to document B1+ field mapping methods in the main text, but it may be something worth spending some time thinking about in a separate issue. How about we say it's fine as-is for this PR and open an issue about documenting these new suffixes in the main specification text?

Reply:
Sounds good to me @tsalo, this would give us more room to come up with a better content organization with other major proposals, such as ASL, are merged.